storage/remote/queue_manager.go

```diff
@@ -1064,7 +1064,7 @@ func sendWriteRequestWithBackoff(ctx context.Context, cfg config.QueueConfig, s
 
 		// If we make it this far, we've encountered a recoverable error and will retry.
 		onRetry()
-		level.Debug(l).Log("msg", "failed to send batch, retrying", "err", err)
+		level.Warn(l).Log("msg", "Failed to send batch, retrying", "err", err)
```

Comment:
Note that failed is capitalized as we normally use sentence case in our logs, and this is how it was before https://github.com/prometheus/prometheus/commit/4eca4dffb89d9da927f1e7884efe85cbd6dc30f3